src/components/BShuttle/BShuttle.vue

```diff
@@ -0,0 +1,404 @@
+<template>
+  <div class='shuttle'>
+
+    <div class='available'>
+      <label id='available-label'>{{ availableLabel }}</label>
+      <ul
+        aria-labeledby='available-label'
+        role='selectbox'
+      >
+        <li
+          v-if="placeholder && (!options || options.length === 0)"
+          :aria-label="placeholder"
+          class='placeholder'
+        >
+        {{ placeholder }}
+        </li>
+        <li
+          v-for="option in availableOptions"
+          :key="'available-option-' + option.id"
+          :id="'available-option-' + option.id"
+          :class="optionsClass(option, 'available')"
+          @click="onOptionClick(option, 'available', 'chosen', $event)"
+          @keyup.space.prevent.stop="onOptionClick(option, 'available', 'chosen', $event)"
+          @keyup.down.prevent.stop="onKeyupDown($event, option, 'available', 'chosen')"
+          @keyup.up.prevent.stop="onKeyupUp($event, option, 'available', 'chosen')"
+          @keyup.16.prevent.stop="onOptionClick(option, 'available', 'chosen', $event)"
+          @keyup.right.prevent.stop="onMoveSelectedClick(selectedOptions, 'available')"
+          @keyup.enter.prevent.stop="onMoveSelectedClick(selectedOptions, 'available')"
+          @keydown.up.prevent.stop
+          @keydown.down.prevent.stop
+          @keydown.space.prevent.stop
+          role='option'
+          tabindex='0'
+        >
+        {{ option.displayText }}
+        </li>
+      </ul>
+    </div>
+
+    <div class='buttons'>
+      <button 
+        @click="onMoveAllOptions('available')"
+        id='all-to-chosen'
+        type='button'
+        >
+        All <i class='fa fa-arrow-right' aria-hidden='true'></i>
+      </button>
+      <button 
+        @click="onMoveSelectedClick(selectedOptions, 'available')"
+        id='selected-to-chosen'
+        type='button'
+        >
+        <i class='fa fa-arrow-right' aria-hidden='true'></i>
```

Comment:
These aren't showing up on the style guide.  I assume font awesome needs to be added to the /src/styleguide.html?  If font awesome is going to be a dependency we should look into getting it included in the buoy build.

Reply:
nope, im waiting for @thedevelopnik 's pr that includes ionicons, and then I will switch them out once I get the new styles from Steve